templates/jobs/confirm_submission.html

```diff
@@ -6,7 +6,7 @@
         <div class="row">
             <div class="col-md-12">
                 {% include 'includes/_info_messages.html' %}
-                <h2>Thank you for adding a job offer!</h2>
+                <h2>Thank you for submission!</h2>
```

Comment:
Either
"Thank you for submitting a job offer!" or
"Thank you for your submission!"